CDF Election Process.md

```diff
@@ -0,0 +1,133 @@
+# CDF Election Process
+
+
+## Election Process
+
+Duration: 1 month
+
+
+### Announcing an Election Cycle
+
+Election cycle consists of 
+
+
+
+* Appointment of Election Officer
+* Initial announcement kicking of the election cycle
+* Nomination period
+* Voting period
+* Announcement of elected individuals
+
+Emails and Slack announcements will be sent out 1 month prior to the nomination period to the relevant subscription lists and channels. If you are not subscribed to a voting class’ email list  or Slack you will not receive these announcements. Please confirm your subscription by visiting [CDF Lists](https://lists.cd.foundation/groups) and reviewing “Your Groups”. Please also make sure your spam filters are set to receive emails from these groups.
+
+Elections will be conducted by CDF Staff. To increase the transparency, elections will be observed by the community member who is appointed as the Election Officer.
+
+
+## Nominations
+
+Nomination period is open for two weeks prior to the election cycle. 
+
+
+### How to Nominate
+
+<span style="text-decoration:underline;">The nomination phase is 2 weeks</span>
+
+Individuals interested in running for these positions or individuals interested to nominate others must record the nominations under corresponding GitHub issues with the name of the candidate, their biography and statement of intent on why they would be a good person to hold the position.
+
+If you are nominating someone other than yourself, please confirm their interest privately BEFORE you nominate them publicly. It is also required that individuals nominated by others accept their nominations directly under GitHub issues. 
+
+
+### Who Can Nominate
+
+Nominations are dependent on the position up for election. Please see the list below  for information on who is eligible to vote and nominate.
+
+
+ #### Governing Board Chair 
+
+* Nominations: Governing Board 
+* Voting: Governing Board
+
+
+#### General Member Representative 1+2 
+
+* Nominations: End User and General Members
+* Voting: End User and General Members
+
+
+#### Committer/Contributor Representative 1+2
```

Comment:
I'm not sure what the 1+2 signifies?

Reply:
it's meant to represent the two positions since there are 2 representatives from the committer/ contributor class. Is there a better way to note that?

Reply:
@justinabrahms I have updated the language to add clarity, please let me know if this is sufficient.